src/Shared-Src/Native/loader.cpp

```diff
@@ -79,13 +73,26 @@ namespace shared {
     };
 
     void Loader::CreateNewSingeltonInstance(ICorProfilerInfo4* pCorProfilerInfo,
-                                            std::function<void(const std::string& str)> logDebugCallback,
-                                            std::function<void(const std::string& str)> logInfoCallback,
-                                            std::function<void(const std::string& str)> logWarnCallback,
-                                            const LoaderResourceMonikerIDs& resourceMonikerIDs,
-                                            WCHAR const * native_profiler_library_filename)
+                                            std::function<void(const std::string& str)> log_debug_callback,
+                                            std::function<void(const std::string& str)> log_info_callback,
+                                            std::function<void(const std::string& str)> log_warn_callback,
+                                            const LoaderResourceMonikerIDs& resource_moniker_ids,
+                                            WCHAR const * native_profiler_library_filename,
+                                            const std::vector<WSTRING>& assembliesToLoad_adDefault_procNonIIS,

```

Comment:
For ex: assembliesToLoad_adDefault_procNonIIS
what "_ad" stands for ?

Reply:
AD stands for App Domain